https://gitlab.gnome.org/GNOME/tracker/-/commit/9df3c725e162

src/tracker-store/meson.build:50:5: ERROR: Function does not take positional arguments.

--- src/tracker-store/meson.build.orig	2020-03-09 23:27:20 UTC
+++ src/tracker-store/meson.build
@@ -47,7 +47,7 @@ dbus_service_file = configure_file(
     install: true,
     install_dir: dbus_services_dir)
 
-i18n.merge_file('tracker-store-desktop-file',
+i18n.merge_file(
     type: 'desktop',
     input: 'tracker-store.desktop.in',
     output: 'tracker-store.desktop',
